index.js

```diff
@@ -1,86 +1,142 @@
-// base-x encoding
-// Forked from https://github.com/cryptocoinjs/bs58
-// Originally written by Mike Hearn for BitcoinJ
-// Copyright (c) 2011 Google Inc
-// Ported to JavaScript by Stefan Thomas
-// Merged Buffer refactorings from base58-native by Stephen Pair
-// Copyright (c) 2013 BitPay Inc
```

Comment:
This has been discarded as the previous implementation is entirely removed,  replaced with the Javascript derivation from bitcoin core.